models/aeif_psc_alpha.cpp

```diff
@@ -94,19 +94,20 @@ nest::aeif_psc_alpha_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bind V to the
```

Comment:
@Silmathoron The English language is a challenging one: "bound" is both past tense of "bind" and a verb in its own right meaning "to confine" or "to limit". The original comment uses "bound" in the second meaning. "Bind" would be incorrect here, since it implied tying the membrane potential to V_peak, instead of just having V_peak as an upper limit. But this discussions shows the need for clearer language, so I would suggest: `// enforce upper limit on V_m`. 

I would not mention "USER DEFINED V_peak_ value in Parameters", since that is clear from the code.